<article id="post-47" class="post-47 page type-page status-publish hentry">
    <header class="entry-header">
        <h1 class="entry-title">Projects</h1>
    </header><!-- .entry-header -->

    <div class="entry-content">
        <p>Below you will find some resources on drugs, drug effects, pharmacological pathways, and genetic
            interactions. All are free and open for academic use (and for most other uses too). Please
            acknowledge and cite our work. If you have any questions please do not hesitate to contact us.
        </p>
        <p>
            You can browse our publicly available code <a href="http://github.com/tatonetti-lab">here</a> 
            and our publicly available resources <a href="http://tatonettilab-resources.s3-website-us-west-1.amazonaws.com">here</a>.
        </p>
    </div><!-- .entry-content -->

</article><!-- #post-## -->




<div class="tlab-resource">

    <h2>OnSIDES, side effects extracted from FDA Structured Product Labels</a></h2>
    <div class="tlab-resource-tags">
    </div>
    <div class="tlab-resource-box">


        <p>OnSIDES (ON label SIDE effectS resource) is the newest member of the NSIDES family. The initial
            release (v01) of the OnSIDES database of adverse reactions and boxed warnings extracted from the FDA
            structured product labels. All labels available to download from DailyMed (<a
                href="https://dailymed.nlm.nih.gov/dailymed/spl-resources-all-drug-labels.cfm" target="_blank"
                rel="noreferrer noopener">https://dailymed.nlm.nih.gov/dailymed/spl-resources-all-drug-labels.cfm</a>)
            as of April 2022 were processed in this analysis. In total 2.7 million adverse reactions were
            extracted from 42,000 labels for just under 2,000 drug ingredients or combination of ingredients.
        </p>



        <p>We created OnSIDES using the ClinicalBERT language model and 200 manually curated labels available
            from Denmer-Fushman et al.. The model achieves an F1 score of 0.86, AUROC of 0.88, and AUPR of 0.91
            at extracting effects from the ADVERSE REACTIONS section of the label and an F1 score of 0.66, AUROC
            of 0.71, and AUPR of 0.60 at extracting effects from the BOXED WARNINGS section.</p>



        <p></p>


        <p>Browse the data at <a href="http://onsidesdb.org">onsidesdb.org</a>.</p>
        <p></p>
        <p>Read more at <a href="http://nsides.io">nsides.io</a>.</p>


        <div class="tlab-citation-box">
            <h3>Cite this resource as</h3>
            <div class="tlab-citation">
                <div class="citation-link">
                    <a href="https://www.medrxiv.org/content/10.1101/2024.03.22.24304724v1"><img draggable="false" role="img"
                            class="emoji" alt="ðŸ”—" src="https://s.w.org/images/core/emoji/14.0.0/svg/1f517.svg"></a>
                </div>
                Tanaka, Y., Chen, H.Y., Belloni, P., Gisladottir, U., Kefeli, J., Patterson, J., Srinivasan, A., Zietz, M., Sirdeshmukh, G., Berkowitz, J., Larow Brown, K., Tatonetti, N. (2024). 
                OnSIDES (ON-label SIDE effectS resource) Database : Extracting Adverse Drug Events from Drug Labels using Natural Language Processing Models.
                medRxiv. 10.1101/2024.03.22.24304724.
            </div>

        </div>
    </div>
</div>
<div class="tlab-resource">

    <h2>Sex-specific side effects (AwareDX)</a></h2>
    <div class="tlab-resource-tags">
        <span class="tlab-resource-tag">#<a rel="tag">algorithm</a></span>
        <span class="tlab-resource-tag">#<a rel="tag">database</a></span>
        <span class="tlab-resource-tag">#<a rel="tag">side effects</a></span>
    </div>
    <div class="tlab-resource-box">

        <p>Adverse drug effects posing sex-specific risks. Risks in this database were predicted by AwareDX to
            publicly available data. Over 20,000 sex risks spanning over 800 drugs and 300 side effects.</p>
        <p>Download the <a href="https://tatonettilab-resources.s3.us-west-1.amazonaws.com/nsides/AwareDX_Data.xlsx.zip">data</a> or browse the&nbsp;<a
                href="https://github.com/tatonetti-lab/sex_risks">repository</a>.</p>


        <div class="tlab-citation-box">
            <h3>Cite this resource as</h3>
            <div class="tlab-citation">
                <div class="citation-link">
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=33179017"><img draggable="false" role="img"
                            class="emoji" alt="ðŸ”—" src="https://s.w.org/images/core/emoji/14.0.0/svg/1f517.svg"></a>
                </div>
                Chandak P, Tatonetti NP. Using Machine Learning to Identify Adverse Drug Effects Posing
                Increased Risk to Women. <i>Patterns (N Y)</i>. 2020;1(7):100108.
                doi:10.1016/j.patter.2020.100108
            </div>

        </div>
    </div>
</div>
<div class="tlab-resource">

    <h2><a href="https://www.tatonettilab.org/offsides/" title="Read more">OffSIDES and TwoSIDES</a></h2>
    <div class="tlab-resource-tags">
        <span class="tlab-resource-tag">#<a href="https://www.tatonettilab.org/tag/algorithm/"
                rel="tag">algorithm</a></span><span class="tlab-resource-tag">#<a
                href="https://www.tatonettilab.org/tag/database/" rel="tag">database</a></span><span
            class="tlab-resource-tag">#<a href="https://www.tatonettilab.org/tag/side-effects/" rel="tag">side
                effects</a></span>
    </div>
    <div class="tlab-resource-box">

        <p>Drug <strong>side effects</strong> and <strong>drug-drug interactions</strong> were mined from
            publicly available data. OffSIDES is a database of drug side-effects that were found, but are not
            listed on the official FDA label.&nbsp;TwoSIDES is the only comprehensive database drug-drug-effect
            relationships. Over 3,300 drugs and 63,000 combinations connected to millions of potential adverse
            reactions.</p>
        <p>Read more and access the data at <a href="http://nsides.io">nsides.io</a>.</p>


        <div class="tlab-citation-box">
            <h3>Cite this resource as</h3>
            <div class="tlab-citation">
                <div class="citation-link">
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22422992"><img draggable="false" role="img"
                            class="emoji" alt="ðŸ”—" src="https://s.w.org/images/core/emoji/14.0.0/svg/1f517.svg"></a>
                </div>
                Tatonetti NP, Ye PP, Daneshjou R, Altman RB. Data-driven prediction of drug effects and
                interactions. <i>Sci Transl Med</i>. 2012;4(125):125ra31. doi:10.1126/scitranslmed.3003377
            </div>

        </div>
    </div>
</div>
<div class="tlab-resource">

    <h2><a href="https://www.tatonettilab.org/family-relationship-and-disease-data/" title="Read more">Family
            Relationship and Disease Data</a></h2>
    <div class="tlab-resource-tags">
        <span class="tlab-resource-tag">#<a 
                rel="tag">algorithm</a></span><span class="tlab-resource-tag">#<a
                 rel="tag">database</a></span>
    </div>
    <div class="tlab-resource-box">

        <p>De-identified family data on over 3,000 conditions at two sites. Data are from approximately 1.5
            million patients across the two sites and all identifying information has been removed. Further,
            ages have been replaced with a random poisson distribution with lambda set to the actual age of the
            patient. Data are compatible with the observation heritability estimation software (<a
                href="https://github.com/tatonetti-lab/h2o">https://github.com/tatonetti-lab/h2o</a>).</p>
        <p>If you would like the <strong>500 significant traits</strong> as reported in Polubriaginof, et al. in
            Cell, go to <a href="riftehr/">this page</a>.</p>
        <p>All code to generate the relationships from hospital data is publicly available in <a
                href="https://github.com/tatonetti-lab/riftehr">our RIFTEHR github</a>.</p>


        <div class="tlab-citation-box">
            <h3>Cite this resource as</h3>
            <div class="tlab-citation">
                <div class="citation-link">
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29779949"><img draggable="false" role="img"
                            class="emoji" alt="ðŸ”—" src="https://s.w.org/images/core/emoji/14.0.0/svg/1f517.svg"></a>
                </div>
                Polubriaginof FCG, Vanguri R, Quinnies K, et al. Disease Heritability Inferred from Familial
                Relationships Reported in Medical Records. <i>Cell</i>. 2018;173(7):1692-1704.e11.
                doi:10.1016/j.cell.2018.04.032
            </div>

        </div>
    </div>
</div>
<div class="tlab-resource">

    <h2><a href="https://www.tatonettilab.org/date/" title="Read more">DATE</a></h2>
    <div class="tlab-resource-tags">
        <span class="tlab-resource-tag">#<a 
                rel="tag">algorithm</a></span><span class="tlab-resource-tag">#<a
                 rel="tag">database</a></span><span
            class="tlab-resource-tag">#<a rel="tag">drug
                discovery</a></span>
    </div>
    <div class="tlab-resource-box">

        <p>Downstream effects of targeted proteins is essential to drug design. We introduce a data-driven
            method named DATE, which integrates drug-target relationships with gene expression, protein-protein
            interaction, and pathway annotation data to connect Drugs to target pAthways by the Tissue
            Expression.&nbsp;Links <b>drugs</b> to <b>tissue-specific target pathways.</b></p>
        <p>467,396 connections for 1,034 drugs and 954 pathways in 259 tissues/cell lines <a
                href="http://tatonettilab-resources.s3-website-us-west-1.amazonaws.com/?p=syspharm/">available</a>.</p>


        <div class="tlab-citation-box">
            <h3>Cite this resource as</h3>
            <div class="tlab-citation">
                <div class="citation-link">
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29920991"><img draggable="false" role="img"
                            class="emoji" alt="ðŸ”—" src="https://s.w.org/images/core/emoji/14.0.0/svg/1f517.svg"></a>
                </div>
                Hao Y, Quinnies K, Realubit R, Karan C, Tatonetti NP. Tissue-Specific Analysis of
                Pharmacological Pathways. <i>CPT Pharmacometrics Syst Pharmacol</i>. 2018;7(7):453-463.
                doi:10.1002/psp4.12305
            </div>

        </div>
    </div>
</div>
<div class="tlab-resource">

    <h2><a href="https://www.tatonettilab.org/gote/" title="Read more">GOTE</a></h2>
    <div class="tlab-resource-tags">
        <span class="tlab-resource-tag">#<a 
                rel="tag">algorithm</a></span><span class="tlab-resource-tag">#<a
                 rel="tag">database</a></span><span
            class="tlab-resource-tag">#<a rel="tag">drug
                discovery</a></span>
    </div>
    <div class="tlab-resource-box">

        <p>G protein-coupled receptors (GPCRs) are central to how cells respond to their environment and a major
            class of pharmacological targets. We developed a data-driven method named GOTE, that connects Gpcrs
            to dOwnstream cellular pathways by the Tissue Expression.&nbsp;Links <b>G-protein coupled
                receptors</b> to <b>tissue-specific molecular pathways</b>.</p>
        <p>93,012 connections for 213 GPCRs and 654 pathways in 196 tissues/cell types <a
                href="http://tatonettilab-resources.s3-website-us-west-1.amazonaws.com/?p=syspharm/">available</a>.&nbsp;Code available <a
                href="http://tatonettilab-resources.s3-website-us-west-1.amazonaws.com/?p=syspharm/">here</a>.</p>


        <div class="tlab-citation-box">
            <h3>Cite this resource as</h3>
            <div class="tlab-citation">
                <div class="citation-link">
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27485444"><img draggable="false" role="img"
                            class="emoji" alt="ðŸ”—" src="https://s.w.org/images/core/emoji/14.0.0/svg/1f517.svg"></a>
                </div>
                Hao Y, Tatonetti NP. Predicting G protein-coupled receptor downstream signaling by tissue
                expression. <i>Bioinformatics</i>. 2016;32(22):3435-3443. doi:10.1093/bioinformatics/btw510
            </div>

        </div>
    </div>
</div>
<div class="tlab-resource">

    <h2><a href="https://www.tatonettilab.org/madss/" title="Read more">MADSS</a></h2>
    <div class="tlab-resource-tags">
        <span class="tlab-resource-tag">#<a 
                rel="tag">algorithm</a></span><span class="tlab-resource-tag">#<a
                 rel="tag">side effects</a></span>
    </div>
    <div class="tlab-resource-box">

        <p class="description">Network analysis framework that identifies adverse event (AE) neighborhoods
            within the human interactome (protein-protein interaction network). Drugs targeting proteins within
            this neighborhood are predicted to be involved in mediating the AE.&nbsp;Links <b>drugs</b> to seed
            sets of <b>proteins</b> and <b>phenotypes</b>, like drug side-effects and diseases.</p>
        <p>A description of the algorithm is <a href="madss/">available
                here</a>.&nbsp;Code in Python available on <a href="http://www.github.com/tal-baum/MADSS">GitHub</a>.
        </p>


        <div class="tlab-citation-box">
            <h3>Cite this resource as</h3>
            <div class="tlab-citation">
                <div class="citation-link">
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25670520"><img draggable="false" role="img"
                            class="emoji" alt="ðŸ”—" src="https://s.w.org/images/core/emoji/14.0.0/svg/1f517.svg"></a>
                </div>
                Lorberbaum T, Nasir M, Keiser MJ, Vilar S, Hripcsak G, Tatonetti NP. Systems pharmacology
                augments drug safety surveillance. <i>Clin Pharmacol Ther</i>. 2015;97(2):151-158.
                doi:10.1002/cpt.2
            </div>

        </div>
    </div>
</div>
<div class="tlab-resource">

    <h2><a href="https://www.tatonettilab.org/venomkb/" title="Read more">VenomKB</a></h2>
    <div class="tlab-resource-tags">
        <span class="tlab-resource-tag">#<a
                rel="tag">database</a></span><span class="tlab-resource-tag">#<a
                 rel="tag">drug discovery</a></span>
    </div>
    <div class="tlab-resource-box">

        <p>The worldâ€™s first comprehensive knowledge base for therapeutic uses of venoms. As of its original
            release, contains 39,000 mined from MEDLINE describing potentially therapeutic effects of venoms on
            the human body.&nbsp;Links <b>venom compounds</b> to <b>physiological effects.&nbsp;</b></p>
        <p>39K venom/effect associations in three databases available
                for download</a>.&nbsp;Code available on <a href="http://www.github.com/jdromano2/venomkb">GitHub</a>.
        </p>


        <div class="tlab-citation-box">
            <h3>Cite this resource as</h3>
            <div class="tlab-citation">
                <div class="citation-link">
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26601758"><img draggable="false" role="img"
                            class="emoji" alt="ðŸ”—" src="https://s.w.org/images/core/emoji/14.0.0/svg/1f517.svg"></a>
                </div>
                Romano JD, Tatonetti NP. VenomKB, a new knowledge base for facilitating the validation of
                putative venom therapies. <i>Sci Data</i>. 2015;2:150065. Published 2015 Nov 24.
                doi:10.1038/sdata.2015.65
            </div>

        </div>
    </div>
</div>
<div class="tlab-resource">

    <h2><a href="https://www.tatonettilab.org/sinatra/" title="Read more">SINaTRA</a></h2>
    <div class="tlab-resource-tags">
        <span class="tlab-resource-tag">#<a
                rel="tag">algorithm</a></span><span class="tlab-resource-tag">#<a
                 rel="tag">database</a></span>
    </div>
    <div class="tlab-resource-box">

        <p>Interspecies, network-based predictions of synthetic lethality and the first genome-wide scale
            prediction of synthetic lethality in humans. Scores were validated against three independent
            databases of synthetic lethal pairs in humans, mouse, and yeast. The original release contains ~109
            million gene pairs with their associated synthetic lethality scores.</p>
        <p>Human synthetic lethal gene pairs available in 3 parts: <a
                href="https://figshare.com/articles/Human_Synthetic_Lethal_Predictions_1_3_/1501103">part 1</a>,
            <a href="https://figshare.com/articles/Human_Synthetic_Lethal_Predictions_2_3_/1501105">part 2</a>,
            and <a href="https://figshare.com/articles/Human_Synthetic_Lethal_Predictions_3_3_/1501115">part
                3</a>. And <a href="https://figshare.com/articles/Mouse_Synthetic_Lethal_Predictions/1519150">mouse</a>
            too.
        </p>


        <div class="tlab-citation-box">
            <h3>Cite this resource as</h3>
            <div class="tlab-citation">
                <div class="citation-link">
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26451775"><img draggable="false" role="img"
                            class="emoji" alt="ðŸ”—" src="https://s.w.org/images/core/emoji/14.0.0/svg/1f517.svg"></a>
                </div>
                Jacunski A, Dixon SJ, Tatonetti NP. Connectivity Homology Enables Inter-Species Network Models
                of Synthetic Lethality. <i>PLoS Comput Biol</i>. 2015;11(10):e1004506. Published 2015 Oct 9.
                doi:10.1371/journal.pcbi.1004506
            </div>

        </div>
    </div>
</div>